app/models/device.py

```diff
@@ -87,3 +88,15 @@ def check_access(self, user: str) -> bool:
             return True
 
         return m.contact_microservice("service", ["check_part_owner"], {"user_uuid": user})["ok"]
+
+    @staticmethod
+    def random(user: str) -> 'Device':
+
+        # Warning this only works for mysql
```

Comment:
`func.random()` should supposedly also work for SQLite

Reply:
Let me leave that here https://stackoverflow.com/questions/60805/getting-random-row-through-sqlalchemy -> func.rand() for mysql but youre right I should also add the option for sqllite for development purposes.

Reply:
Have you tried it? https://stackoverflow.com/questions/60805/getting-random-row-through-sqlalchemy#comment68799157_60815

Reply:
func.rand() supports mysql and sqllite ? ...